### PR TITLE
41 webhelp

### PR DESCRIPTION
**Pull request by ismirlia**
### What are the issues this pull addresses (issue numbers / links)?
#41 

### Did you add tests to cover your changes (yes/no)?
no
### Did you update the documentation accordingly (yes/no)?
no
### Did you read the CONTRIBUTING document (yes/no)?
yes
### Summary
Added web help button to metal GUI.

### Details and comments

Screen Shot 2021-09-29 at 1 58 36 PM

### CODE REVIEW

File: qiskit_metal/_gui/main_window.py [hunk 1]
```diff
@@ -202,6 +214,11 @@ def create_build_log_window(self, _=None):
         """"Handles click on Build History button."""
         self.gui.gui_create_build_log_window()
 
+    @slot_catch_error()
+    def open_web_help(self, _=None):
+        """"Handles click on Build History button."""
+        webbrowser.open('https://qiskit.org/documentation/metal/', new=1)
```
Comment (zlatko-minev) [inline qiskit_metal/_gui/main_window.py:220]: Nice!

**Comment by zlatko-minev** [comment]
Hi @ismirlia , thank you for adding the help button to the gui that links out to the qiskit metal documentation webpage. This is very nice and really helpful. It looks good to me, we may just want to check the gui runs ok with the new UI files compiled.

**Review by zlatko-minev — APPROVED** [review]

**Review by marcolincs — COMMENTED** [review]
I believe there is no need for the "examples" folder. Can you please remove it? For reference, if you name the folder `_sandbox_`, it will be ignored by github, so it can safely remain in your local disk only.

File: qiskit_metal/_gui/main_window.py [hunk 2]
```diff
@@ -114,6 +115,17 @@ def show_renderer_q3d(self):
 
     def delete_all_components(self):
         """Delete all components."""
+
+        # yesnoMsg = QMessageBox()
+        # yesnoMsg.setWindowTitle("Delete all components?");
+        # yesnoMsg.setText("Are you sure you want to clear all Metal components?");
```
Comment (marcolincs) [inline qiskit_metal/_gui/main_window.py:121]: I believe this is a test you were playing with. Can you please remove it form the code in your next commit?
Cheers

Comment (ismirlia) [reply qiskit_metal/_gui/main_window.py]: Sorry I committed again took and took that stuff out. I thought I had already cleaned up the code before committing it the first time, so I made some sort of mistake in the process.

**Comment by zlatko-minev** [comment]
This is very nice, I wonder if we could just post a screenshot of what the GUI looks like now with the latest install that Axel made? 


Other than that I'm good ￼￼

**Comment by marcolincs** [comment]
This is what I see. Is this as-expected?
image

**Comment by ismirlia** [comment]
yes that looks right

**Review by marcolincs — APPROVED** [review]
LGTM

**Pull request merged**